### ISSUE TITLE
Fix/add references to tokens in exercises before 3

### ISSUE THREAD

**Issue by ajmcmiddlin** [qfpl/elm-workshop#3]
It seems exercise 3 expects us to have the following fields:

```elm
type alias Model =
    { ...
    , loginError : Maybe String
    , loginToken : Maybe String
    , registerError : Maybe String
    , registerToken : Maybe String
    ...
    }
```

However, at the beginning of the exercise I ended up with the following:

```elm
type alias Model =
    { ...
    , backendError : Maybe String
    , token : Maybe String
    ...
    }
```

In exercises preceding 3, we should make it clear which fields we expect and why. This will ensure that examples refer to fields that students already have, and that consistent names are used to avoid confusion.

**Comment by ajmcmiddlin** [comment]
Part of this can probably be solved by changing `backendError` to `loginError` in the `Main.elm` that students start with, rather than asking them to change it.

**Comment by ajmcmiddlin** [comment]
Fixed by #2
